chain/jsonrpc/build_errors_schema.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 cd "${0%/*}/../.." # ensure we're in the workspace directory
-rm target/rpc_errors_schema.json
+rm -f target/rpc_errors_schema.json
```

Comment:
What's changed here? Could this file now not be present? What step writes to it?

Reply:
Can we just remove this line all together? Either way it's fine.

Reply:
I’ll keep it for now. This entire logic will need to be migrated over to a nextest test as part of the next steps in migrating over from buildkite, and I’m not really prepared to investigate any potential fallout from removing this line right now.